zad4/app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,9 @@
+class TeamsController < ApplicationController
+  def index
+    @captains=Player.where(captain: true).to_a
+    @players=Player.where(captain: false).to_a
+    @teams=@players.count/@captains.count
```

Comment:
`Stwórz widok zawodników, który zestawi ich w losowe, 11 osobowe drużyny. ` Jak będzie 1000 zawodników i tylko 2 z nich będzie kapitanami, to nieźle tu się wszystko rozjedzie :D

Reply:
Poza tym należy w rails uważać na metody count, size i length :) Ich zastosowanie zależy od danej sytuacji:

https://mensfeld.pl/2014/09/activerecord-count-vs-length-vs-size-and-what-will-happen-if-you-use-it-the-way-you-shouldnt/

Reply:
I w tej linii kodu przy braku żadnych kapitanów wychodzi dzielenie przez 0 ;)